### ISSUE TITLE
Apple GSX Integration (IRP / AASP Program)

### ISSUE THREAD

**Issue by Jonmarr95** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1650]
Hello, as RepairDesk integrates with more suppliers I wanted to see if there are any plans or interest in integrating with Apple? We are an Apple partner and I would love to be able to automate parts of inventory and purchase orders. Currently the only competitor to offer GSX integration is a website called Fixably. We are willing to help with testing in any way.

**Comment by Uzair-RepairDesk** [comment]
Hey @Jonmarr95 

Thank you for your post. We would love to connect with you to learn the workflow of your IRP/AASP business. We are already exploring GSX to integrate it with RepairDesk so it will be helpful for us to learn more about your needs. 
Can you please confirm your business name associated with RepairDesk? Is it "Tech Bros"? 
We will reach out to you to discuss this further. Thanks

Looking forward to helping you grow

Regards,
Uzair

**Comment by antoinegali** [comment]
Great idea! 

Samsung GSPN integration would be cool too. We're going to be approved for the "Samsung IRP" in the following months and I would really love to have it integrated.

**Comment by jphansendk** [comment]
GSX Yes please :-)

**Comment by Jonmarr95** [comment]
@Uzair-RepairDesk Yes, our company is Tech Bros. Please connect and we’ll help however we can.
